cmd/pgsink/cmd/app.go

```diff
@@ -154,6 +162,31 @@ func Run() (err error) {
 		)
 	}
 
+	// If we're given a Sentry DSN, we'll want to initialise the client. Sentry clients can
+	// be acquired via the sentry.Hub helpers, which manage a collection of clients on
+	// behalf of many concurrent processes.
+	//
+	// https://pkg.go.dev/github.com/getsentry/sentry-go
+	if *sentryDSN != "" {
```

Comment:
Cribbed this from another project which has sentry. Not a bad idea though, keeping it!